.env.example

```
@@ -4,13 +4,11 @@ GITHUB_TOKEN=your_github_token
 
-KV_REST_API_TOKEN=your_kv_token
-KV_REST_API_URL=https://your.api/url
 WEBHOOKS_REPLICATOR_TOKEN=
 
 # advanced (optional)
 # These variables are kept for linting compatibility only
 # They should remain empty in a regular usage
 GH_APP_ID=
 GH_APP_INSTALLATION_TOKEN=
-GH_APP_PRIV_KEY_BASE64=
+GH_APP_PRIV_KEY_BASE64=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)